Script para descomentar artigos que recibimos

deivisCode opened the pull request:
(véxase #51)

deivisCode committed:
Script para descomentar artigos que recibimos

arebla commented:
@deivisCode Ben pensado. A única suxerencia que faría é a posibilidade de meter o nome do documento como un parámetro, tipo que o script execútese como `./descomenta.sh nome_artigo.tex`.

Unha posible implementación sería cos seguintes cambios:
```sh
#!/bin/bash

entrada="$1"
nome_ficheiro="$(basename "$entrada" .${entrada##*.})"
extension="${entrada##*.}"

saida="${nome_ficheiro}_descomentado.${extension}"
cat "$entrada" | \
  sed '/^\%\%\%\:/d'                 | \
  sed '/^\\documentclass{revista}/d' | \
  sed '/^\\begin{document}/d'        | \
  sed '/^\\end{document}/d'          | \
  sed '/^\\begin{refsection}/d'      | \
  sed '/^\\end{refsection}/d'        > "$saida""
```

deivisCode committed:
Erros e documentaciono

- Estaba borrando o `refsection`, o cal hai que manter
- Agora poden descomentarse varios arquivos de golpe
- Engadín algúns '%%%:' máis no artigo simplificado para que ao borralos
  non nos quedemos con demasiadas liñas baleiras
- Comentei como funciona todo
